backend/src/main.py

```diff
@@ -490,18 +491,13 @@ def addEmployee():
     if existing_project==None:
         return jsonify({'error':'Project with ID: {} Does not Exist !'.format(project_code)})
 
-    project_list = existing_project.split(",")
-    for i in project_list:
-        if i=="":
-            project_list.remove(i)
+    project_list=helper1(existing_project)
+    removenull(project_list)
     if project_code in project_list:
         return jsonify({'error':'Employee is already working in the project'})
     project_list.append(project_code)
         
-    out_resource =''
-    for i in project_list:
-            out_resource = i + "," + out_resource
-    existing_project.project_code = out_resource[:-1]
+    existing_project.project_code =helper2(project_list)
```

Comment:
give some name for the helper2 function like "listToString"/"StringToList"
the remove null should not  be a separate function but should be a part of the helper2 function